Description: Fix barbican-api.conf as parsable
 Upstream file isn't parsable, this patch makes it so.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-06-13

--- barbican-2015.1.0.orig/etc/barbican/barbican-api.conf
+++ barbican-2015.1.0/etc/barbican/barbican-api.conf
@@ -35,10 +35,9 @@ max_allowed_request_size_in_bytes = 1000
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
 # Uncomment this for local dev, putting db in project directory:
-#sql_connection = sqlite:///barbican.sqlite
 # Note: For absolute addresses, use '////' slashes after 'sqlite:'
 # Uncomment for a more global development environment
-sql_connection = sqlite:////var/lib/barbican/barbican.sqlite
+sql_connection = sqlite:////var/lib/barbican/barbicandb
 
 # Period in seconds after which SQLAlchemy should reestablish its connection
 # to the database.
